deno.jsonc

```diff
@@ -1,12 +1,14 @@
 {
   "fmt": {
     "files": {
-      "include": ["src"]
+      "include": ["src"],
+      "exclude": ["src/schema/slack/functions/_scripts/functions.json"]
```

Comment:
Why do we want to exclude this? The file isn't checked into git, so this would just be running `fmt` locally right?